pkg/git/git.go

```diff
@@ -394,3 +395,57 @@ func (r *Repo) DescribeTag(rev string) (string, error) {
 
 	return strings.TrimSpace(status.Output()), nil
 }
+
+// Pull retrieves the latest remote content and merges is into the branch
+func (r *Repo) Pull(from string) error {
+	worktree, err := r.inner.Worktree()
+	if err != nil {
+		return err
+	}
+
+	return worktree.Pull(&git.PullOptions{
+		RemoteName: DefaultRemote,
+		Auth:       r.auth,
+		Progress:   os.Stdout,
+	})
+}
+
+func (r *Repo) Commit() error {
```

Comment:
Needs a method description. Same for the other methods below.

Reply:
Added them :+1: